model/gen-ai/registry.yaml

```diff
@@ -56,6 +56,18 @@ groups:
               stability: development
               value: "aws.bedrock"
               brief: '[AWS Bedrock](https://aws.amazon.com/bedrock)'
+            - id: alibaba.dashscope
```

Comment:
is it possible to document the whole convention, not just the constant name? It's the best practice:

https://github.com/open-telemetry/semantic-conventions/blob/e7eb01668175e11a9cf3b7adb57b38603505a448/docs/how-to-write-conventions/README.md?plain=1#L181-L182